.github/workflows/lint.yml

```diff
@@ -36,11 +36,8 @@ jobs:
       - name: Determine changed packages
         id: packages
         run: |
-          FILTER=$(turbo run lint --dry-run=json --filter '...[HEAD^]' | jq -e '.packages | contains(["//"])' > /dev/null && echo '' || echo '--filter ...[HEAD^]')
-          PACKAGES=$(sh -c "turbo run lint --dry-run=json $FILTER" \
-            | jq '.tasks[]' \
-            | jq 'select(.task == "lint")' \
-            | jq --compact-output --slurp '{ package: [.[].package] | unique, include: [( .[] | {package: .package, directory: .directory })] | unique }')
+          PACKAGES=$(turbo query 'query { affectedPackages { items { name path } } }' \
```

Comment:
shouldn't this filter for packages that have the lint task?

Reply:
There is no `lint` task per package. I have no idea why it worked before, probably because we have a top-level `lint` task. It just assumes that we want to lint every package which has been changed. This is also why I had to exclude `//` in `jq`.